### DISCUSSION TITLE
EvCharger ModbusRTU V2.8 documentation

### DISCUSSION THREAD

**Discussion by romankysely** [PatrikTrestik/homeassistant-solax-http#22] [Ideas]
Zdravím Vás Patriku a díky moc za váš příspěvek s touto integrací. Možná se vám bude hodit (snad poslední) verze popisu Modbusu
[EvCharger ModbusRTU V2.8 (1).pdf](https://github.com/PatrikTrestik/homeassistant-solax-http/files/14743367/EvCharger.ModbusRTU.V2.8.1.pdf)
.

**Comment by PatrikTrestik** [comment]
Diky. Uloženo.
Máte nějaký zdroj dokumentace pro SolaX? Sháním jakýkoliv dokument k HTTP API, ale zatím jsem nic nenašel.

**Comment by PatrikTrestik** [comment]
Aktuální verze je 3.5, kterou mám.

Uzavírám.
